### ISSUE TITLE
Configuration for comparing dates with correct format - row hash audit

### ISSUE THREAD

**Issue by PraveenAkunuru** [GoogleCloudPlatform/professional-services-data-validator#534]
For the row hash audits, We are trying to do some audit between TD and BigQuery. The row hash audit failed, due to a difference in the default cast of dates in TD and BQ. One Example -:

In TD - 1989/08/06

In BQ – 1989-08-06

Is there a way to fix/ cast these at the tool level, rather than at the source/ target level?

**Comment by nehanene15** [comment]
This would likely involve adding support for the FORMAT function in Teradata/BQ
```
SELECT
   CAST( (CAST (F1 AS FORMAT 'YYYY-MM-DD')) AS CHAR(10))
   FROM INTDAT;
```

[Teradata docs link](https://docs.teradata.com/r/kmuOwjp1zEYg98JsB8fu_A/_WAvgD2iMJtE9l~GlnyWOA)

**Comment by nehanene15** [comment]
Closed with #747
